docs/content.rst

```diff
@@ -382,6 +382,40 @@ to allow linking to both generated articles and pages and their static sources.
 
 Support for the old syntax may eventually be removed.
 
+Including other files in your pages
+-----------------------------------
+Both Markdown an reStructuredText syntaxes provide mechanisms for this.
+
+There are some examples for **reStructuredText** using `reStructuredText include`_:
+
+    .. code-block:: none
+
+        .. include:: file.rst
+
+Including a fragment of a file delimited by two identifiers, highlighted as C++ (slicing based on line numbers is also possible):
+
+    .. code-block:: none
```

Comment:
Would `.. code-block:: rst` work to highlight this? Not sure about Sphinx in particular, but Pygments at least understand this as reStructuredText. 

Same in other cases.

Reply:
👍 That works !
Strange because it hadn't found it on http://pygments.org/languages/
But that definitely performs some code highlighting :)

Reply:
ReST is listed there (and `rest` works too I think) :)